### PR TITLE
Update Make commands & fix inconsistent setup

### PR DESCRIPTION
**Pull request by btobin**
- Add back additional setup commands to fix inconsistent install of Gems and NPM packages
- Add additional PLATFORMS to Gemfiile.lock for both ARM/x86 linux
- Clean up and better document local Makefile test commands from host

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,16 +43,18 @@ down:
...
-# Enter the shell of the running dev container
+# Enter the shell of the *running* dev container
 shell:
 	docker-compose exec site bash
 
-# Build the dev container from scratch, overcoming 
-# a funky (documented) error with installed packages 
-# that may hav something to do with use of a volume
+# Build the dev container from scratch.
+# Runs packages installs a second time for both Gems and NPM to
```
Comment (RedBrogdon) [inline Makefile:51]: nit: "packages' installs" or "package installs?"

File: Makefile [hunk 2]
```diff
@@ -64,45 +67,62 @@ serve:
...
-# local flutter repo via the volume, so this may have 
-# unexpected consequences on your local setup.
+# Convenience command to run inside of a container to 
+# switch the channel of the flutter submodule. 
+# NOTE that changes will be reflected in your local flutter repo 
 # Usage: `make switch-channel CHANNEL=<channel>`
 switch-channel:
-	
-	docker-compose exec site cd flutter && \
+	cd flutter && \
```
Comment (RedBrogdon) [inline Makefile:87]: Out of curiosity, why is `flutter channel ${CHANNEL}` not used here? Are you looking for finer grained access to additional branches (beyond master/dev/beta/stable, I mean)?

File: Makefile [hunk 3]
```diff
@@ -42,16 +43,18 @@ down:
...
-# Build the dev container from scratch, overcoming 
-# a funky (documented) error with installed packages 
-# that may hav something to do with use of a volume
+# Build the dev container from scratch.
+# Runs packages installs a second time for both Gems and NPM to 
+# overcome inconsistent bugs with missing packages at runtime.
 setup:
 	make clean
 	docker-compose build site
+	docker-compose run --rm site bundle install
```
Comment (RedBrogdon) [inline Makefile:56]: For the record, I was unable to build the site without these lines. Glad to see them back.

**Review by RedBrogdon — APPROVED** [review]

**Review by johnpryan — APPROVED** [review]
This is working for me

**Pull request merged**